deploy/docker-compose-full.yml

```diff
@@ -38,6 +38,9 @@ services:
       - ../configs:/opt/notary/configs
     environment:
       - PROFILE
+      - IROHA_HOST=d3-iroha
+      - IROHA_PORT=50051
+      - CLASS=registration.eth.relay.DeployRelayMain
```

Comment:
you don't need to specify main class. just apply 'application' plugin and specify variable `mainClassName` inside gradle.
Then, if you use docker plugin, it will work "as is"

Reply:
We can't, because our services are not split 
So we have one application but many classnames that we have to run

Reply:
you need to create separate gradle.build for each of projects then.

see https://github.com/soramitsu/BCA-Sora-Middleware/

Reply:
Especially pay attention to https://github.com/soramitsu/BCA-Sora-Middleware/blob/master/settings.gradle